infra/README.md

```diff
@@ -0,0 +1,55 @@
+# DeltaNYC Infrastructure
+
+Infrastructure is maintained using;
+
+* [Terraform](https://www.terraform.io/)
+* [Cloud-Init](http://cloudinit.readthedocs.io/en/latest/index.html)
+
+
+# To the droplet's ip address
+
+```bash
+$ curl -s -X GET -H "Content-Type: application/json" -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" "https://api.digitalocean.com/v2/droplets" | jq '.[] | .[] | select(.name=="delta_lookup_tool") | .networks.v4 | .[]'
+```
+
+We're currently using the DO ubuntu image, which allows ssh directly to the root account.
+
+```bash
+ssh -i ~/.ssh/id_rsa_delta root@{public_id}
+```
+
+The key to the machine is maintained by Chris Henry
+
+
+# Configuration
+
+### Google Maps API Key.
+
+The key in use was generated from Chris Henry's Google API Console, but any key
+with Google Maps permissions can be used.
+
+# Setup steps
+
+We should be using luigid, but I can't quite figure out what it's supposed to be
+doing.
```

Comment:
Ahh, got it. that's super helpful. So with the workers, do they execute a set number of jobs and then exit? I observed the one worker do a few jobs, and then exit with code 137 (?).

Next round, I'll definitely play around with using luigi to totally automate the process.